### PR TITLE
gbesjes.wxMUN version 0.4

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17882)

### CODE REVIEW

**Review by alpdikmen — COMMENTED** [review]
.

File: manifests/g/gbesjes/wxMUN/0.4/gbesjes.wxMUN.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: gbesjes.wxMUN
+PackageVersion: 0.4
+PackageLocale: en-US
+Publisher: gbesjes
+PackageName: wxMUN
```
Comment (OfficialEsco) [inline manifests/g/gbesjes/wxMUN/0.4/gbesjes.wxMUN.locale.en-US.yaml:8]:
```suggestion
Publisher: 'Geert-Jan Besjes'
Author: gbesjes
PackageName: wxMUN
```

File: manifests/g/gbesjes/wxMUN/0.4/gbesjes.wxMUN.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: gbesjes.wxMUN
+PackageVersion: 0.4
```
Comment (OfficialEsco) [inline manifests/g/gbesjes/wxMUN/0.4/gbesjes.wxMUN.installer.yaml:5]:
```suggestion
PackageVersion: 0.40
```

File: manifests/g/gbesjes/wxMUN/0.4/gbesjes.wxMUN.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: gbesjes.wxMUN
+PackageVersion: 0.4
```
Comment (OfficialEsco) [inline manifests/g/gbesjes/wxMUN/0.4/gbesjes.wxMUN.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.40
```

File: manifests/g/gbesjes/wxMUN/0.4/gbesjes.wxMUN.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: gbesjes.wxMUN
+PackageVersion: 0.4
```
Comment (OfficialEsco) [inline manifests/g/gbesjes/wxMUN/0.4/gbesjes.wxMUN.yaml:5]:
```suggestion
PackageVersion: 0.40
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image

**Pull request closed without merging**